### DISCUSSION TITLE
第4节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#249] [笔记]
第4节课笔记(3班)

**Comment by 443752009** [comment]
http://t.csdnimg.cn/TgTd6

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/10/mmtutorial-4-note/

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/677124128

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135580086

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/PXrFdh5aaobSozx4rmscuMVunih?from=from_copylink

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]XTuner 单卡低成本微调实战](https://zhuanlan.zhihu.com/p/677766199)

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/677761706

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/ckwgnwahtbxgoy2l?singleDoc# 《XTuner 大模型单卡低成本微调实战笔记》

**Comment by Sampson425** [comment]
笔记
https://blog.csdn.net/m0_49908454/article/details/135581858

**Reply by Sampson425** [reply]
https://blog.csdn.net/m0_49908454/article/details/135688928

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/678950457

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135863318
